nltk/classify/rte_classify.py

```diff
@@ -70,8 +70,8 @@ def __init__(self, rtepair, stop=True, lemmatize=False):
         self.hyp_words = set(self.hyp_tokens)
 
         if lemmatize:
-            self.text_words = set(lemmatize(token) for token in self.text_tokens)
```

Comment:
What I mean is that:
* the parameter to this class is broken
* you renamed a public function
* so you broke something that worked for the sake of fixing something that no one uses

So the right fix, assuming someone *does* want to use the parameter, is to rename the parameter or give the external function two names.